src/main/java/org/owasp/wrongsecrets/StartupListener.java

```diff
@@ -1,46 +0,0 @@
-package org.owasp.wrongsecrets;
-
-import java.util.Arrays;
-import java.util.stream.Collectors;
-import lombok.experimental.UtilityClass;
-import lombok.extern.slf4j.Slf4j;
-import org.jetbrains.annotations.NotNull;
-import org.springframework.boot.context.event.ApplicationEnvironmentPreparedEvent;
-import org.springframework.context.ApplicationEvent;
-import org.springframework.context.ApplicationListener;
-
-/** Helps application startup and breaks nicely if K8S_ENV is wrong. */
-@Slf4j
-public class StartupListener implements ApplicationListener<ApplicationEvent> {
-
-  @Override
-  public void onApplicationEvent(final @NotNull ApplicationEvent event) {
-    if (event instanceof ApplicationEnvironmentPreparedEvent envEvent) {
-      if (!StartupHelper.passedCorrectEnv(envEvent.getEnvironment().getProperty("K8S_ENV"))) {
-        log.error(
-            "K8S_ENV does not contain one of the expected values: {}.",
```

Comment:
see earlier comment how do we make it crash nicely and readable? this is ostly for contributors and first time runners to make sure they do it right.